lineapy/plugins/task.py

```diff
@@ -187,6 +187,54 @@ class TaskSerializer(Enum):
     # TODO: lineapy.get and lineapy.save
 
 
+def render_task_definitions(
+    task_defs: Dict[str, TaskDefinition],
+    pipeline_name: str,
+    task_serialization: Optional[TaskSerializer],
+    task_name_fn: Callable = lambda task_def: task_def.function_name,
+    function_decorator_fn: Callable = lambda task_def: "",
+    user_input_variables_fn: Callable = lambda task_def: ", ".join(
+        task_def.user_input_variables
+    ),
+    typing_blocks_fn: Callable = lambda task_def: task_def.typing_blocks,
+    pre_call_block_fn: Callable = lambda task_def: task_def.pre_call_block,
+    call_block_fn: Callable = lambda task_def: task_def.call_block,
+    post_call_block_fn: Callable = lambda task_def: task_def.post_call_block,
+    return_block_fn: Callable = lambda task_def: "",
+    include_imports_locally: bool = False,
```

Comment:
We dont need to add it, loaded_input_variables is used by `render_task_io_serialize_blocks` so it doesnt need a function since its behavior is parametrized by TaskSerializer.